frontend/src/component/AddTopicPage.tsx

```diff
@@ -31,9 +32,18 @@ export default function AddTopicPage() {
         )
     }, [topicData, setTopicData, selectedSize, setSelectedSize])
 
+    function displayError() {
+        if (errorMsg !== "") {
+            return (
+                <Alert severity="warning">{errorMsg}</Alert>
+            )
+        }
+    }
+
 
     return (
         <div className="addTopicPage">
+            {displayError()}
```

Comment:
Maybe rename the method to something like `ifErrorMsgDisplayError` so that it is immediately clear reading this line that it is a conditional render.